packages/untp-test-suite-mocha/scripts/generate-inference-bundle.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/**
+ * Generate Inference Bundle
+ *
+ * This script scans the inferences directory for .n3 files and generates
+ * a TypeScript file with their contents embedded for browser use.
+ * This allows the browser bundle to access inference rules without needing
+ * Node.js filesystem modules.
```

Comment:
Nice - so our standard n3 queries are included in the bundle and therefore available to the client. Regarding @ashleythedeveloper 's question about extensibility, I don't see any reason why (later) a user couldn't include their own extension queries to the client browser (or for standard extensions, that we couldn't include them in the bundle either), do you?